scripts/checkoutForm.js

```diff
@@ -0,0 +1,41 @@
+const form = document.getElementById('checkout-form');
+//form.setAttribute('id', 'checkout-form');
+//const fields = ['Name', 'Email', 'Street Address', 'City', 'State', 'Zip code', 'First time adopter?', 'Pickup location'];
+
+//for (let i = 0; i < fields.length; i++) {
+  //const label = document.createElement('label');
+  //label.textContent = fields[i];
+  //label.setAttribute('for', fields[i]);
+
+  //const input = document.createElement('input');
+  //input.setAttribute('type', 'text');
+  //input.setAttribute('id', fields[i]);
+
+  //form.appendChild(label);
+  //form.appendChild(input);
+//}
+
+const submitButton = document.createElement('button');
+submitButton.setAttribute('type', 'submit');
+submitButton.textContent = 'submit';
+
+form.addEventListener('submit', function(event) {
```

Comment:
Your checkout button does not appear to be logging out the form values. I'm thinking maybe conflicting with some of the jQuery code